docs/docs/account/user-permissions.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: User Permissions
+description: GrowthBook's User Permissions
+sidebar_label: User Permissions
+slug: user-permissions
+---
+
+GrowthBook supports user roles and permissions at both the global and project level. This allows you to control who can access GrowthBook and what they can do.
+
+When inviting a new user to your organization, you can give the user a global role, which defines their permissions across all projects. If you are enrolled in a `Pro` or `Enterprise` account, you
+have the added ability to give a user project-level roles.
+
+:::note
+
+For instance, you can set a user's global role as `Collaborator`, which allows them to view features and experiements, as well as add
+comments and contribute ideas, and then give them an `Experimenter` role for a particular project, which allows them to create and run experiments, but only for the specified project.
+
+:::
+
+Below are the roles currently available in GrowthBook and their associated permissions.
+
```

Comment:
Also for docs changes I often run them by Chat-GTP asking them to rephrase for clarity.  For the most part I like its suggestions slightly better than what you have:

GrowthBook provides user roles and permissions at both the global and project level. This feature enables you to manage who can access GrowthBook and their level of access.

When inviting a new user to your organization, you can assign a global role, which defines their permissions across all projects. If you have a `Pro` or `Enterprise` account, you can also assign project-level roles to users.

:::note
For example, you can assign a user the global role of `Collaborator`, allowing them to view features and experiments, add comments, and contribute ideas. You can then assign them an `Experimenter` role for a specific project, which allows them to create and run experiments, but only for that project.
:::

The table below lists the roles available in GrowthBook and their associated permissions.